receiver/kubeletstatsreceiver/kubelet/metadata.go

```diff
@@ -28,10 +28,12 @@ type MetadataLabel string
 
 const (
 	MetadataLabelContainerID MetadataLabel = conventions.AttributeContainerID
+	MetadataLabelVolumeType  MetadataLabel = labelVolumeType
 )
 
 var supportedLabels = map[MetadataLabel]bool{
 	MetadataLabelContainerID: true,
+	labelVolumeType:          true,
```

Comment:
shouldn't it be `MetadataLabelVolumeType`?

Reply:
Fixed.